docker-compose.yml

```diff
@@ -159,6 +162,173 @@ services:
             -  INTERNAL_OBSERVATION_PORTAL_API_URL=http://127.0.0.1:8000
         entrypoint: /entrypoint.sh
 
+    minio1:
+        image: minio/minio:RELEASE.2021-06-17T00-10-46Z
+        volumes:
+            - data1-1:/data1
+            - data1-2:/data2
+        expose:
+            - "9000"
+        environment:
+            MINIO_ROOT_USER: minio
+            MINIO_ROOT_PASSWORD: minio123
+            MINIO_REGION_NAME: minioregion
+        command: server http://minio{1...4}/data{1...2}
+        healthcheck:
+            test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+            interval: 30s
+            timeout: 20s
+            retries: 3
+    minio2:
+        image: minio/minio:RELEASE.2021-06-17T00-10-46Z
+        volumes:
+            - data2-1:/data1
+            - data2-2:/data2
+        expose:
+            - "9000"
+        environment:
+            MINIO_ROOT_USER: minio
+            MINIO_ROOT_PASSWORD: minio123
+            MINIO_REGION_NAME: minioregion
+        command: server http://minio{1...4}/data{1...2}
+        healthcheck:
+            test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+            interval: 30s
+            timeout: 20s
+            retries: 3
+    minio3:
+        image: minio/minio:RELEASE.2021-06-17T00-10-46Z
+        volumes:
+            - data3-1:/data1
+            - data3-2:/data2
+        expose:
+            - "9000"
+        environment:
+            MINIO_ROOT_USER: minio
+            MINIO_ROOT_PASSWORD: minio123
+            MINIO_REGION_NAME: minioregion
+        command: server http://minio{1...4}/data{1...2}
+        healthcheck:
+            test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+            interval: 30s
+            timeout: 20s
+            retries: 3
+    minio4:
+        image: minio/minio:RELEASE.2021-06-17T00-10-46Z
+        volumes:
+            - data4-1:/data1
+            - data4-2:/data2
+        expose:
+            - "9000"
+        environment:
+            MINIO_ROOT_USER: minio
+            MINIO_ROOT_PASSWORD: minio123
+            MINIO_REGION_NAME: minioregion
+        command: server http://minio{1...4}/data{1...2}
+        healthcheck:
+            test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+            interval: 30s
+            timeout: 20s
+            retries: 3
```

Comment:
probably a dumb question but why are there 4 minios?

Reply:
Minio requires a distributed erasure code setup in order to enable versioning for buckets, which I believe is required by the science archive. And 4 is the minimum number of servers for a distributed minio setup. Elisabeth posted some details in a comment on the pivotal story.

Reply:
That's right. I think this might warrant a comment seeing as others will probably have the same question.

Reply:
Explanatory comments added.